webui/react/src/pages/WorkspaceDetails.tsx

```diff
@@ -75,9 +76,9 @@ const WorkspaceDetails: React.FC = () => {
   );
   const pageRef = useRef<HTMLElement>(null);
   const workspaceId = workspaceID ?? '';
-  const id = parseInt(workspaceId);
+  const id = Number(workspaceId);
```

Comment:
i think `parseInt` isnt the right func cuz this makes `1abc` -> `1`. switched to `Number`